test/NUnit.Portable.Agent.Tests/NUnitPortableDriverTests.cs

```diff
@@ -122,7 +114,7 @@ public void RunTestsAction_AfterLoad_ReturnsRunnableSuite()
             _driver.Load(_frameworkAssembly, _mockAssembly, _settings);
             var result = XmlHelper.CreateXElement(_driver.Run(null, EMPTY_FILTER));
 
```

Comment:
@rprouse - I didn't initially realise that wrapping the results up how I currently have done would change the XML, and return a test-run rather than a test-suite. Would this cause any problems for dotnet-test-nunit, should it be done differently?